.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,6 +1,9 @@
 ---
 name: Bug report
 about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
```

Comment:
not sure how this is useful

Reply:
Edited and removed those, not sure how they got suggested in the first place.